src/scss/attendee/breakpoint.scss

```diff
@@ -1,5 +1,5 @@
-$sm: 0576px; // mobile
```

Comment:
In many languages (including JS of various versions) a leading 0 indicates octal. I found this confusing so got rid of it.

Reply:
So, removing the line because the removal of the `0` wasn't going to solve the octal problem?